src/html_files/system_info.ts

```diff
@@ -1,4 +1,43 @@
 let got_system_info_data = false;
+let system_info_rules = {
+    data_type: "system_info",
+    pretty_name: "System Info",
+    rules: [
+        {
+            name: "System Name",
+            func: function (ruleOpts: RuleOpts) {
+                let os_version = get_data_key(ruleOpts.data_type, "OS Version");
+                return is_unique_map(ruleOpts.per_run_data) && is_unique_map(os_version);
+            },
+            good: "Same OS across runs.",
+            bad: "Different OS and/or version across runs."
+        },
+        {
+            name: "Total CPUs",
+            func: function (ruleOpts: RuleOpts) {
+                return is_unique_map(ruleOpts.per_run_data);
+            },
```

Comment:
The cpu utilization rules are returning arrays of Findings, and this is returning boolean. Lock this function signature down or it will be a nightmare to maintain having to support a wide variety of return types.

Reply:
Okay. I'm changing it so that only Finding should be returned.